llvm/test/Transforms/Util/add-TLI-mappings.ll

```diff
@@ -2,9 +2,8 @@
 ; RUN: opt -vector-library=MASSV -passes=inject-tli-mappings -S < %s | FileCheck %s  --check-prefixes=COMMON,MASSV
```

Comment:
Despite the fact that MASSV, LIBMVEC or Accelerate libraries do not check if the target is correct, I think it is a good practice to specify it in the RUN line, otherwise it may lead to incorrect behaviour if something in the compiler change in the future.
So I think here we would need to add:  -mtriple=powerpc64-unknown-linux-gnu here

and  -mtriple=x86_64-unknown-linux-gnu in the X86 and Accelerate case